cli/templateedit.go

```diff
@@ -57,6 +59,7 @@ func templateEdit() *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&name, "name", "", "", "Edit the template name")
+	cmd.Flags().StringVarP(&displayName, "display-name", "", "", "Edit the template display name")
```

Comment:
Should we add this to `template create` as well?

Reply:
As agreed [here](https://github.com/coder/coder/issues/3321#issuecomment-1307760364), we will go with `templates edit` for now. I don't mind extending the `create` command if there is a need for this option.